internal/app/secretless/listeners/sshagent/listener.go

```diff
@@ -3,25 +3,19 @@ package sshagent
 import (
 	"fmt"
 	"log"
-	"net"
 	"strconv"
 
 	"golang.org/x/crypto/ssh/agent"
 
 	"github.com/conjurinc/secretless-broker/internal/pkg/util"
 	"github.com/conjurinc/secretless-broker/pkg/secretless/config"
 	plugin_v1 "github.com/conjurinc/secretless-broker/pkg/secretless/plugin/v1"
-	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/go-ozzo/ozzo-validation"
```

Comment:
Third-party imports should be between local and built-in ones (even though it wasn't you that added it originally)